test/reporter/writers.go

```diff
@@ -1,4 +1,4 @@
-package e2e
```

Comment:
This may have been the cause? It was in reporter directory, but under the name e2e? In any case, I prefer it being at the top level of test

Reply:
Nah, tried that too, was still showing no logs, the real trick was moving the modules out of test/e2e.